amz_review_scraper/amzscraper.py

```diff
@@ -14,46 +14,28 @@
 }
 
 
-# def find_attribute(soup, key, html_tag, attrs):
-#    temp_json = {}
-#    print(attrs)
-#    for divs in soup.findAll(html_tag, attrs=attrs):
-#        temp_json = divs[key]
-#        print("temp =" + temp_json)
-#    return temp_json
-# return next((divs[key] for divs in soup.findAll(html_tag, attrs=attrs):
-#        if divs[key] is not null))
+def find_attribute(soup, search_tag, key, html_tag, attrs):
+    for divs in soup.findAll(html_tag, attrs=attrs):
```

Comment:
Im still trying to figure out how to sub out "for divs" to be "for variable passed into the function" @davidbegin

Reply:
shoot, i figured it out, I was thinking that the "divs" was important to the searching but its only to match up with the return divs[key] so i dont even need to change that the search spans etc.  but I should change the name so its not confusing